libs/sdk-core/src/chain.rs

```diff
@@ -124,6 +125,7 @@ pub(crate) fn get_utxos(address: String, transactions: Vec<OnchainTx>) -> Result
 #[derive(Clone)]
 pub(crate) struct MempoolSpace {
     pub(crate) base_url: String,
+    pub(crate) fallback_base_url: Option<String>,
```

Comment:
Is there any point `fallback_base_url` being optional if the Config `mempoolspace_fallback_url` is not optional?

Reply:
Good point, fixed.